cls/SourceControl/Git/Utils.cls

```diff
@@ -322,6 +347,69 @@ ClassMethod SwitchBranch(targetBranchName As %String) As %Status
     quit $$$OK
 }
 
+ClassMethod PreSync() As %String
+{
+    set uncommittedFilesWithAction = ##class(SourceControl.Git.WebUIDriver).UncommittedWithAction().%Get("user")
```

Comment:
Nitpick: I would move this UncommittedWithAction() method into the Utils class because it's less tidy to have Utils referencing the WebUI code.